Specs/Scene/ModelExperimental/ModelExperimentalSpec.js

```diff
@@ -459,6 +460,31 @@ describe(
       });
     });
 
+    it("initializes with model matrix", function () {
+      const translation = new Cartesian3(10, 0, 0);
+      const transform = Matrix4.fromTranslation(translation);
+      const modelMatrix = new Matrix4();
+      Matrix4.multiplyTransformation(Matrix4.IDENTITY, transform, modelMatrix);
+
+      return loadAndZoomToModelExperimental(
+        {
+          gltf: boxTexturedGlbUrl,
+          upAxis: Axis.Z,
+          forwardAxis: Axis.X,
+          modelMatrix: modelMatrix,
+        },
+        scene
+      ).then(function (model) {
+        const sceneGraph = model.sceneGraph;
```

Comment:
I would also add something like
`expect(model.modelMatrix).not.toBe(transform)` (and similar for the computed model matrix below) just as a sanity check that the constructor is cloning the value.